### DISCUSSION TITLE
posts/2023/30f06a/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#14] [Announcements]
# posts/2023/30f06a/

2023 年大三的我，脑子里在想些什么呢？

https://blog.gzti.me/posts/2023/30f06a/

**Comment by Cyy0808** [comment]
So we beat on, boats against the current, borne back ceaselessly into the past.

**Comment by Tokisakix** [comment]
just feel it

**Comment by Elapsedf** [comment]
或许终有一天能够找寻到问题的答案，但人或许总归要前进

**Comment by pandacase** [comment]
fine
